src/rt/thread.rs

```diff
@@ -126,10 +128,17 @@ impl Thread {
 
     pub(crate) fn set_terminated(&mut self) {
         self.state = State::Terminated;
+    }
+
+    pub(crate) fn drop_locals(&mut self) -> Box<dyn std::any::Any> {
```

Comment:
Any reason this doesn't just return a `Vec<LocalValue>`, or is this just to hide the actual type from calling code so that all it can do is drop it?

Reply:
Also: might be good to have a comment here explaining why this doesn't actually drop anything :)

Reply:
It was a private type, so I opted to hide it... I didn't think deeply about it.